cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -175,6 +178,12 @@
 
     private static final String NAME_SUFFIX = "-kafka";
 
+    private static final String SECRET_JMX_SECRET_SUFFIX = NAME_SUFFIX + "-jmx";
```

Comment:
why `SECRET` two times?

Reply:
Whoops should be named `KAFKA_JMX_SECRET_SUFFIX`